docs/dev-guide/plugin-dev.md

```diff
@@ -432,6 +432,49 @@ module.exports.defaultModes = {
 
 This is because the command's expected mode needs to be known before loading environment variables, which in turn needs to happen before loading user options / applying the plugins.
 
+### Local plugins
+
+If you need access to the plugin API in your project and don't want to create a full plugin for it, you can use the `vuePlugins.service` option in your `package.json` file:
+
+```json
+{
+  "vuePlugins": {
+    "service": ["my-local-plugin.js"]
+  }
+}
+```
+or
+```json
+{
+  "vuePlugins": {
+    "service": {
+      "myPlugin": "my-local-plugin.js"
+    }
+  }
+}
+```
+When using the object configuration, key `myCommands` becomes your local plugin `name` and the plugin options will be directly accessible as the third argument in `PluginAPI` if the plugin `name` corresponds to the key defined in `pluginOptions`.
```

Comment:
There's no context for `myCommands` here.

Reply:
You are correct.